provision/overlay/bin/audioctl

```diff
@@ -94,7 +92,6 @@ case "$VERB" in
 	mute)
 		case "$VOLUME_MANAGER" in
 			amixer)
-				amixer -q set Master 100%-
```

Comment:
This behavior was desired. Pulseaudio is weird - I've hit edge cases where simply muting an output device does not cause sound to stop playing.

Reply:
Ah, this makes the `unmute` case more difficult.